src/javascript/app_2/App/Components/Form/TimePicker/dialog.jsx

```diff
@@ -0,0 +1,71 @@
+import classNames   from 'classnames';
+import React        from 'react';
+import { localize } from '_common/localize';
+import { toMoment } from 'Utils/Date';
+
+function selectOption(type, value, props, is_enabled = true) {
```

Comment:
Think it's better if you move this to the `Duration` component? The function is only related to that component and that would follow the same style as our other components